packages/zapp/console/src/components/Literals/LiteralMapViewer.tsx

```diff
@@ -20,13 +22,21 @@ export const LiteralMapViewer: React.FC<{
   className?: string;
   map: LiteralMap | null;
   showBrackets?: boolean;
-}> = ({ className, map, showBrackets = false }) => {
+  showJson?: boolean;
+}> = ({ className, map, showBrackets = false, showJson }) => {
   if (!map) {
     return <NoDataIsAvailable />;
   }
 
   const commonStyles = useCommonStyles();
   const { literals } = map;
+
+  if (!Object.keys(literals).length) {
```

Comment:
nit: looks like duplicate logic with 
https://github.com/flyteorg/flyteconsole/blob/2e9725ea0fc510a0bf6b5a07b8346bb9055aa9d6/packages/zapp/console/src/components/Literals/LiteralMapViewer.tsx#L40-L52

Reply:
that's because the mapcontent logic is intended to go away